src/views/components/trace/trace-search.vue

```diff
@@ -236,6 +236,10 @@
     }
 
     private created() {
+      const endpoint: any = this.$route.query.endpointname || this.endpointName;
```

Comment:
Would it be better to write this way?

```js
this.endpointName = this.$route.query.endpointname ? this.$route.query.endpointname : this.endpointName
this.traceId = this.$route.query.traceid? this.$route.query.traceid: this.traceId
```

Reply:
@QHWG67 I think the above one is more clear than `||`.

Reply:
@x22x22 @wu-sheng per your requested, they are updated. Pls check.

Notes:
by default, this.$route.query.endpointname is the string array (it means the URL parameter may have duplicated key name, this should not be happen normally), so it cannot directly be "this.endpointName = this.$route.query.endpointname", then get error as below, so here has to use toString() and get all values of one key name, user may find their url problem and adjust url accordingly base on result. This update is verified by me before commiting.
252:5 Type 'string | string[]' is not assignable to type 'string'.
  Type 'string[]' is not assignable to type 'string'.
    250 |
    251 |     private created() {
  > 252 |       this.endpointName = this.$route.query.endpointname? this.$route.query.endpointname : this.endpointName;

Reply:
Make sense. In theory, people could set duplicated keys in the URL parameters. It is a list, not a map. I think `toString` is fine. 

I am going to merge this. Could you submit the submodule sync to main repo after I am done?